Assembly-CSharp/ModLoader.cs

```diff
@@ -23,15 +23,28 @@ namespace Modding
     [PublicAPI]
     internal static class ModLoader
     {
+        [Flags]
+        public enum ModLoadState
+        {
+            NotStarted = 0,
+            Started = 1,
+            Preloaded = 2,
+            Loaded = 4,
+        }
+
+        public static ModLoadState modLoadState = ModLoadState.NotStarted;
```

Comment:
This shouldn't be camel case, would also abbreviate to LoadState or even just State.